src/main/io/gps.c

```diff
@@ -417,9 +419,40 @@ void gpsInitNmea(void)
                if (!atgmRestartDone) {
                    atgmRestartDone = true;
                    serialPrint(gpsPort, "$PCAS02,100*1E\r\n");  // 10Hz refresh rate
-                   serialPrint(gpsPort, "$PCAS10,0*1C\r\n");    // hot restart 
+                   serialPrint(gpsPort, "$PCAS10,0*1C\r\n");    // hot restart
+               }
+
+               //NMEA custom commands
```

Comment:
It may be better to implement new GPS_STATE_CUSTOM before GPS_STATE_INITIALIZED. It will improve command sequencing (comments above)